README.md

```diff
@@ -1 +1,129 @@
-# OOP-school-library
\ No newline at end of file
+<a id="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+<details>
+  <summary><h1>📗 Table of Contents</h1></summary>
+  <ul>
+    <li><a href="#about-project">📖 About the Project</a>
+      <ul>
+        <li><a href="#built-with">🛠 Built With</a></li>
+        <li><a href="#tech-stack">Tech Stack</a></li>
+        <li><a href="#key-features">Key Features</a></li>
+      </ul>
+    </li>
+    <li><a href="#getting-started">💻 Getting Started</a>
+      <ul>
+        <li><a href="#setup">Setup</a></li>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#install">Install</a></li>
+        <li><a href="#usage">Usage</a></li>
+        <li><a href="#run-tests">Run tests</a></li>
+        <li><a href="#deployment">Deployment</a></li>
```

Comment:
![image](https://github.com/EvansSnave/OOP-school-library/assets/93039359/9a5ace71-25be-41b3-a7e9-11454a5a2422)

- [ ] Great work so far but it would be nice if you fix the highlighted links above 👀 because these links are broken 😨 Kindly fix them or remove them if you don't have any dedicated section(s) for these broken links 🤔 to make your readme descriptive and professional 👍